<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Job</class>
 <widget class="QDialog" name="Job">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="PushJob1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushJob2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushJob4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushJob5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushJob3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushJob6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushJob7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushJob8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Job8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUp8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Level8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="coins">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Coins :</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="coinsToUp8">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CoinsToUp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CoinsToUp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Income</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Income</string>
   </property>
  </widget>
  <widget class="QLabel" name="income5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="income6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="income7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="income8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>179</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="income3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>139</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="income1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="income4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="income2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Score :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpCity">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="levelOfCity">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>35</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </widget>
  <widget class="QLabel" name="cityTimer">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>35</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushLuck">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Luck</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
